manifests/4.10/image-references

```diff
@@ -11,3 +11,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-metallb:4.10
+  - name: metallb-frr-rhel8
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-metallb-frr:4.10
```

Comment:
Shouldn't it be origin-frr?
Although I cannot find either in Quay.io:
- https://quay.io/openshift/origin-metallb-frr
- https://quay.io/openshift/origin-frr

Reply:
this is where we are mirroring the image to: https://github.com/openshift/release/blob/0c552acc78842033cab91d9a9c24f4eddb731cea/core-services/image-mirroring/openshift/mapping_origin_4_10#L183

Reply:
And that's a placeholder, CI replaces it with the last image from the CI registry iirc.